test/end-to-end/pages/deleteBlueprint.js

```diff
@@ -25,17 +25,17 @@ module.exports = class deleteBlueprint extends MainPage {
 
     // ---- Page element selector ---- //
     // Page Title
-    this.labelPageTitle = `${this.headerElement} h4[class="modal-title"]`;
+    this.labelPageTitle = `${this.headerElement} [data-id="myModalLabel"]`;
```

Comment:
Related to a previous comment above, if we keep the heading element as it originally was:
`<h4 className="modal-title" id="myModalLabel">`
Then we can use the id selector here:
this.labelPageTitle = `${this.headerElement} #myModalLabel`;